google-cloud-pub-sub-grpc/src/main/mima-filters/1.1.x.backwards.excludes/io.grpc.backwards.excludes

```diff
@@ -0,0 +1,21 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+# Upgrading to io.grpc 1.64 caused these issues
```

Comment:
when adding mima filters, it'd be good to document why the detected incompatibilities are harmless.

In this case: these classes are in a package called `.internal.`, so it looks like these methods are not intended to be called from 'outside', and indeed it seems unlikely that they are - so the change indeed seems acceptable to me.